TaskCalendar.tsx

```
@@ -1,8 +1,8 @@
 import React, { useState, useMemo } from 'react';
 import { Calendar, momentLocalizer } from 'react-big-calendar';
 import moment from 'moment';
-import { Task } from '../types';
-import { useTaskStore } from '../store/taskStore';
+import { Task } from '@/types';
+import { useTaskStore } from '@/store/taskStore';
 import 'react-big-calendar/lib/css/react-big-calendar.css';
 import { Calendar } from 'lucide-react';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Calendar'. Consider to delete it or rename it.

'Calendar' is defined here:


(lint/suspicious/noRedeclare)